### ISSUE TITLE
authentication issues

### ISSUE THREAD

**Issue by derekcfoley** [custom-components/alexa_media_player#1364]
I find the whole integration setup confusing with regard to authentication. Is there no way it can be simplified? After an update to HA a month ago, my Alexa Media Player stopped working, I've tried dozens of times to reauthenticate, reinstall the Alexa Media Player integration and turn on and off Amazon 2FA etc, the whole authentication dialog during setup seems to go in circles - the information that shows at the top doesn't help, neither does the dozens of options, cookie import, tickboxes etc. Really haven't a clue about how to set it up properly as it keeps failing... 

Might delete the integration. Why is this so damned difficult and complicated? Why can't it "just work" with my own devices  out of the box so to speak? Pun intended ;)

**Comment by derekcfoley** [comment]
When I get to this stage - 

![image](https://user-images.githubusercontent.com/25664126/132503253-d36bc809-30ae-4839-9c69-f8708ca8a7d5.png)

I get the push message to my mobile, I "approve" it on the mobile, 
Then I click submit on the bottom of the above - the integration configuration doesn't do anything... it just refreshes, doesn't go anywhere!

**Comment by alandtse** [comment]
Enable 2FA or use the Login Proxy. 2FA is mandatory for support.

**Comment by derekcfoley** [comment]
Tried that. It keeps showing "Unknown Error" in Red (see image)

![image](https://user-images.githubusercontent.com/25664126/132912940-6c36feb0-1b4c-47cc-85e2-e4c91b768749.png)

**Comment by derekcfoley** [comment]
Also notice the "second authenticator app" is shown here, that was just confirmed by this proceedure... 

![image](https://user-images.githubusercontent.com/25664126/132913250-4cff646d-4ace-49d0-98eb-0ef1cda7487a.png)

**Comment by derekcfoley** [comment]
I'm in the UK. Something is clearly broken with the Authentication process and the integration now since its release.

**Comment by derekcfoley** [comment]
Also when I close the authentication process window as it won't work to try again, I then see this when I click "configure" on the integration as I did before. 
![image](https://user-images.githubusercontent.com/25664126/132913538-7fbb240b-65df-49b6-8aba-5c72c777d5ea.png)

**Comment by derekcfoley** [comment]
Ps. I'm using the Nabu Casa cloud service with Homeassistant, if that makes any difference.

**Comment by derekcfoley** [comment]
After a restart of HomeAssistant... I click the configure button again... now it shows this: 

![image](https://user-images.githubusercontent.com/25664126/132914575-8918b932-c60e-4d3b-a807-d68e970e48e6.png)

I enter my 2FA code from the Google Authenticator app running on my phone, and the captcha code... and the dialog refreshes showing "There was a problem" 

never works.

**Comment by alandtse** [comment]
Please fill out the bug report template.

**Comment by derekcfoley** [comment]
Figured it out. Basically after I cancelled my Nabu Casa subscription a while back before reinstating it, I had changed my browser Homeassistant link to my internal IP address. This is why every time I tried to setup the integration afterwards - its always failed. 

This messed up the Amazon / Alexa Media Player Integration 2FA authorisation process. Changing it back to the Nabu Casa provided URL for access made the config and 2FA process work finally. All is well.
